### DISCUSSION TITLE
Windows can't access this disc,The disc might be corrupt

### DISCUSSION THREAD

**Discussion by Niyi-FDI** [iterate-ch/cyberduck#14645] [Q&A]
I have tested Cyber Duck and everything worked well.

I decided to buy Mountain Duck because I like the idea of mounting folders.

However, when I mounted one of my S3 bucket, I was not able to open the My S3 bucket in Mountain Duck it gave the following error...

"Windows can't access this disc

The disc might be corrupt. Make sure that the disc
uses a format that Windows recognizes. If the disc is unformatted, you need to format it before using it."

Strangely, the other  S3 bucket  open fine.

Also, if I go to Cyber Duck, I am able to open all my S3 bucket.

![image (1)](https://github.com/iterate-ch/cyberduck/assets/107966657/f4d5f939-749c-4149-af23-86bab3894ba8)

MD version 4.11.2
Windows OS
Please help.

**Comment by dkocher** [comment]
Let me know if you can reproduce with the latest release version **4.13.8**.

**Reply by Niyi-FDI** [reply]
Yes, I am experiencing the same error on 4.13.8

**Comment by tatze96** [comment]
Please enable [debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the described behavior, and send us any available *mountainduck.log* and *mountainduck-\*.log.zip* files for further investigation to support@mountainduck.io. Ensure to include the ticket reference #14645.

You can ZIP the log files or share them through a cloud storage of your choice if they are too big for sending.
